docs/azure_data_studio/README_FOR_MARKETPLACE.md

```diff
@@ -101,3 +101,56 @@ For more information see the [Code of Conduct FAQ][conduct-FAQ] or contact [open
 [conduct-FAQ]: http://opensource.microsoft.com/codeofconduct/faq/
 [conduct-email]: mailto:opencode@microsoft.com
 [conduct-md]: https://github.com/PowerShell/vscode-powershell/blob/master/CODE_OF_CONDUCT.md
+
+## SQL PowerShell Examples
+In order to use these examples (below), you need to install the SqlServer module from the [PowerShell Gallery](https://www.powershellgallery.com/packages/SqlServer).
+
+<pre><code>Install-Module -Name SqlServer -AllowPrerelease</code></pre>
+
+In this example, we use the Get-SqlInstance cmdlet to Get the Server SMO objects for ServerA & ServerB.  The default output for this command will include the Instance name, version, Service Pack, & CU Update Level of the instances.
+
+<pre><code>Get-SqlInstance -ServerInstance ServerA, ServerB</code></pre>
+Here is a sample of what that output will look like:
+<pre><code><# Sample Output #>
+Instance Name             Version    ProductLevel UpdateLevel
+-------------             -------    ------------ -----------
+ServerA                   13.0.5233  SP2          CU4
+ServerB                   14.0.3045  RTM          CU12</code></pre>
+
+In this example, we will do a 'dir' (alias for Get-ChildItem) to get the list of all SQL Server instances listed in your Registered Servers file, and then use the Get-SqlDatabase cmdlet to get a list of Databases for each of those instances.
```

Comment:
backticks for markdown formatting.

```suggestion
In this example, we will do a `dir` (alias for `Get-ChildItem`) to get the list of all SQL Server instances listed in your Registered Servers file, and then use the `Get-SqlDatabase` cmdlet to get a list of Databases for each of those instances.
```

Reply:
Sure thing.  I can take care of that right now.